Makefile

```diff
@@ -82,4 +82,8 @@ compact:
 			echo "-> Generate tar for $${GOOS}/$${GOARCH}..."; \
 			tar cfz dist/gsh-$${GOOS}-$${GOARCH}.tar.gz README.md LICENSE -C dist/$${GOOS}/$${GOARCH} .; \
 		done; \
-	done
\ No newline at end of file
+	done
```

Comment:
I also missed a function to test this new `_test.go` file. 

Something that would eventually run this: ```go test -v ./...```

Reply:
Great, I make it at #52.